client/header/activity-panel/activity-card/style.scss

```diff
@@ -227,6 +232,13 @@
 	.woocommerce-activity-card__header {
 		margin-bottom: $gap-small;
 	}
+
+	&.actioned {
+		height: 0;
+		opacity: 0;
+		padding: 0;
+		transition: opacity 1s, height 0s 1s, padding 0s 1s;
```

Comment:
That's my personal opinion, but I felt the animation to be a bit slow. :turtle: What would you think about making it a bit faster to `0.6s` or something like this?

Reply:
Done in cf8ea88.